.github/ISSUE_TEMPLATE/meeting_setup.md

```diff
@@ -40,3 +40,23 @@ to maximise the notification period and increase attendance but reminders can be
       Slack](https://nhsrcommunity.slack.com/).
 + [ ] **Sheffield Digital Slack** : If you are a member you can announce the event on the [Sheffield Digital Slack](https://sheffield.digital/slack).
 + [ ] **Royal Statistical Society Sheffield Local Group** Contact the [RSS Sheffield Local](https://rss.org.uk/membership/rss-groups-and-committees/groups/sheffield/) group and ask them to disseminate details of the meeting (contacts are currently [Esther Herbert](mailto:e.herbert@sheffield.ac.uk) but may change over time).
+
+## During the Event
+
+During the event, a couple of tasks are needed and it would be helpful if organisers can share tasks. If you'd like to help out, tick the task and put you name next to it.
+
++ [ ] Update Schedule Slides: Modify the schedule and add the speaker's slides if they send it before hand
++ [ ] Pre Set up: Arrive to the room a bit earlier to make sure the projectors are all working
++ [ ] Facilitator: Introduce speakers, facilitate Q&A, and mention feedback form and volunteer form.
++ [ ] Hybrid: Log into the Google Meet and make sure the slides and voice are in order for online attendees. Also monitor the chat and relay the questions to the speakers
++ [ ] Media: Snap a couple of pictures of the speakers with their consent to upload on the meetup page.
+
+
+## Post Event
+
++ [ ] **Add material**: If the speakers had slides, ask them for the slides and add them under "SheffieldR.github.io/posts
+/date of meet up/"
++ [ ] **Send post event message**: From the meetup page, go to the event page, at the right hand side click on "Orgnaizer Tools" then on "Contact Attendees".
+  + [ ] Choose: Members who RSVPed Yes for this Meetup
```

Comment:
I think it would be worthwhile sharing this with everyone on the list, that way those who couldn't make it see the material and we might entice more people to attend in future.

What do you think?

Reply:
yes I think you're right